### DISCUSSION TITLE
Fit data with asymmetric error bars on both axes

### DISCUSSION THREAD

**Discussion by Elisa-Visentin** [scikit-hep/iminuit#942] [Q&A]
Hi, I have to fit a set of data points whose error bars are asymmetric on both axes (e.g., Y1+0.2-0.3, X1+0.1-0.2, Y2+0.1-0.5.....). Can `iminuit` face this task? How? 
Thank you!

**Comment by HDembinski** [comment]
iminuit currently does not handle this case, because it is ill-defined, see section 6.3 in https://arxiv.org/pdf/1905.12362.pdf
